Backpack-SwiftUI/CardCarousel/Classes/BPKCardCarousel.swift

```diff
@@ -66,23 +68,58 @@ internal struct InternalCardCarousel<Content: View>: View {
         content: [Content],
         currentIndex: Binding<Int>
     ) {
+        self.isIPad = UIDevice.current.name.hasPrefix("iPad")
+        self.size = size
         self._currentIndex = currentIndex
-        self.cardWidth = size.width * 0.8
+        self.cardWidth = isIPad ? (size.width * 0.8) * 0.4 : size.width * 0.8
         self.cardCount = content.count
         self._currentInternalIndex = State(initialValue: cardCount + 1 + currentIndex.wrappedValue)
-        self.content = content + content
+        
+        if isIPad && UIDevice.current.orientation.isLandscape {
+            self.content = content
+        } else {
+            self.content = content + content
+        }
+        
         focusOnCard = .init(currentInternalIndex)
     }
     
     var body: some View {
+        if isIPad && UIDevice.current.orientation.isLandscape {
+            carouselIPadLandscape
+        } else if isIPad {
+            carouselIPadPortrait
+        } else {
+            carousel
+        }
+    }
+    
+    private var carouselIPadLandscape: some View {
+        HStack(spacing: .xxl) {
+            ForEach(Array(content.enumerated()), id: \.offset) { _, card in
+                card
```

Comment:
Should we support a carousel on landscape as well if there are more cards?

Reply:
We discussed about this, landscape will not be implemented now and wil be added later on after discussions with design, or even contributed when there's a usecase that needs that.